webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -359,30 +367,35 @@ export class TargetAggregatesService {
     return aggregates.find(aggregate => aggregate.id === targetId);
   }
 
-  getCurrentTargetDoc(contact?) {
-    return this.ngZone.runOutsideAngular(() => this._getCurrentTargetDoc(contact));
+  getTargetDocs(contact, userFacilityId:string[]|string|undefined, userContactId:string|undefined):Promise<any[]> {
```

Comment:
```suggestion
  getTargetDocs(contact, userFacilityIds:string[]|undefined, userContactId:string|undefined):Promise<any[]> {
```
Can it only receive an array of strings? Then `isUserFacility` only checks the array with includes. The consumer can pass the array with 1 id if necessary, `getTargetDocs([id])`.  Or the user settings service can return an array of facilities always.

Reply:
I'll check if we ever get the single facility id as a string. I thought this was possible. Is it not?

Reply:
I'm enforcing the the array in the reducer now. Previously it could show up as array or string.